vuepress/docs/next/docs/reference/backup-restore.md

```diff
@@ -43,46 +42,46 @@ kubectl cp quickstart-deployment:/tmp/entando-data.tar.gz entando-data.tar.gz
-3. For Bundle DB schemas, different methods are required for auto-provisioned and external databases.
+3. For bundle DB schemas, different methods are required for auto-provisioned and external databases.
 
     A. For auto-provisiond DBMS schemas:  
              1. When the provisioning is complete, scale down the plugin deployment to 0. Restore the source data with the database-native backup/restore tools and then scale up the deployment.  
              2. In case of GCG-generated Spring Boot plugins, the plugin's application will autonomously apply the Liquibase data migration scripts.   
              3. In order to connect to an internal auto-provisioned DBMS, use kubectl port-forward against the bundle DB.   
 
-    B. For External DBMS schemas:  
+    B. For external DBMS schemas:  
        1. Export all the EntandoDeBundle CRs from the source instance and restore them to the target instance.  
-       2. Then create the restored instance DB schema by making a copy of the source instance DB schema with the DB native backup/restore tools.  
-       3. Copy DB credential secrets from the original instance into the restored instance.  
+       2. Then create the restored instance DB/schema by making a copy of the source instance DB/schema with the DB-native backup and restore tools.  
+       3. Copy DB credential Secrets from the original instance into the restored instance.  
        4. Reinstall the bundles.   
 
 
 ### Private Repository Bundles
-If a bundle originated from a private repository in the original instance, the secrets, configurations, etc. will need to be replicated in the new instance. 
-* For private Docker registry bundles, see [Install Bundle from a Private Image Registry](https://developer.entando.com/v7.1/tutorials/curate/bundle-private-images.html) tutorial. 
-* For bundles from a private Git repository, see [Install Bundles from a Private Git Repository](https://developer.entando.com/v7.1/tutorials/curate/private-git-repo.html)  tutorial.
+If a bundle originated from a private repository in the source instance, the Secrets and configurations will need to be replicated in the new instance. 
+* For private Docker registry bundles, see the [Install Bundle from a Private Image Registry](https://developer.entando.com/v7.1/tutorials/curate/bundle-private-images.html) tutorial. 
+* For bundles from a private Git repository, see the [Install Bundles from a Private Git Repository](https://developer.entando.com/v7.1/tutorials/curate/private-git-repo.html) tutorial.
 
 ## Configure the Service Address
-If the restored instance service address will remain the same as the original, and needs to be on the same cluster, the two ingresses can collide and end with failure for the restored instance.
+If the restored instance is deployed into the same cluster with the identical domain name as the original, the ingresses can collide and result in failure for the restored instance.
```

Comment:
@nshaw, think you should read this whole doc but this part especially because I didn't get the full picture from PJ

Reply:
Done, maybe we can discuss briefly today